python-avd/tests/pyavd/j2filters/test_list_compress.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import pytest
+from pyavd.j2filters.list_compress import list_compress
+
+
+class TestListCompressFilter:
+    @pytest.mark.parametrize(
+        "invalid_list_to_compress, error_msg",
+        [
```

Comment:
Assign this test list to a variable and replace list with variable name.

Reply:
Use tuple as we are doing https://github.com/aristanetworks/avd/pull/4101/files#diff-b2720323f846110d5d8206ea52bb472950761954caa6f6ec4929836bbf038499R11

Reply:
Updated it!